modules/eks/README.md

```diff
@@ -20,22 +20,24 @@
-| node\_pools\_launch\_kind     | Which kind of node pools to create. Valid values are: launch\_templates, launch\_configurations, both.                                                 | `"launch_templates"` |    no    |
-| resource\_group\_name         | Resource group name where every resource will be placed. Required only in AKS installer (*)                                                            | `""`                 |    no    |
-| ssh\_public\_key              | Cluster administrator public ssh key. Used to access cluster nodes with the operator\_ssh\_user                                                        | n/a                  |   yes    |
-| subnetworks                   | List of subnets where the cluster will be hosted                                                                                                       | n/a                  |   yes    |
-| tags                          | The tags to apply to all resources                                                                                                                     | `{}`                 |    no    |
+| Name                                     | Description                                                                                                                                            | Default              | Required |
+|------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------|:--------:|
+| cluster\_endpoint\_public\_access        | Indicates whether or not the Amazon EKS public API server endpoint is enabled.                                                                         | `false`              |    no    |
+| cluster\_endpoint\_public\_access\_cidrs | List of CIDR blocks which can access the Amazon EKS public API server endpoint.                                                                        | `["0.0.0.0/0"]`      |    no    |
+| cluster\_log\_retention\_days            | Kubernetes Cluster log retention in days. Defaults to 90 days.                                                                                         | `90`                 |    no    |
+| cluster\_name                            | Unique cluster name. Used in multiple resources to identify your cluster resources                                                                     | n/a                  |   yes    |
+| cluster\_version                         | Kubernetes Cluster Version. Look at the cloud providers documentation to discover available versions. EKS example -> 1.16, GKE example -> 1.16.8-gke.9 | n/a                  |   yes    |
+| dmz\_cidr\_range                         | CIDR blocks which can access the Amazon EKS private API server endpoint.                                                                               | `"0.0.0.0/0"`        |    no    |
```

Comment:
I'd suggest keeping this description similar to the `cluster_endpoint_public_access_cidrs` one instead of two different descriptions.

Reply:
The type are different. For public one, I used a `list(string)` as also done by AWS. 
For the existent `dmz_cidr_range` I avoided to change the type to avoid some branking change. So it expect a single CIDR.